pkg/registry/core/next/context.go

```diff
@@ -1,4 +1,6 @@
-// Copyright (c) 2020 Cisco Systems, Inc.
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+//Copyright (c) 2020 Cisco Systems, Inc.
```

Comment:
Indenting, Also I think it should start with Cisco, since it was first.

Reply:
Thanks, fixed